docs/glossary.md

```diff
@@ -55,20 +55,41 @@
 ## T
 
 - [Token Allowance](#token-allowance)
+- [Triplet](#triplet)
 
 ## W
 
 - [Weibar](#weibar)
 
 ---
 
-## Account Aliases
+## Aliases
 
-**Account Aliases**: These are identifiers which can be protobuf serialized keys of ED25519 or ECDSA
-Keys or 33 byte EVM address values. The "triplet" form (e.g. 0.0.18273892) is the simpler and easier
-to "type". An alias is not intended to be user-friendly, but instead to offer a smart-contract
-friendly identifier. See https://hips.hedera.com/hip/hip-32 and https://hips.hedera.com/hip/hip-583
-for more information.
+**Alias**: A value that can be associated with an account, and used in some circumstances instead
+of the account id [triplet](#triplet).
+
+Most often used to indicate a 20-byte value that can be used in smart contracts to refer to a
+Hedera account.
+
+All accounts will have a (derived) account num alias, but all accounts _may_ have a single
```

Comment:
It feels like it would be clearer if we said _may also_ instead of _may_?  I don't feel strongly about this - ignore if you disagree.

Reply:
Leave it?  It sounds to me like the way RPCs and stuff are written ....

Reply:
If you want to more closely match `RFC` language, the language for alias in the [pending update to the protobuf files](https://github.com/hashgraph/hedera-protobufs/blob/e4dfc42ab7aaaf755731e3db037c91c1a3736777/services/basic_types.proto#L133) might be helpful.
(This is part of a, draft for now, [giant PR](https://github.com/hashgraph/hedera-protobufs/pull/388) to update all that text).

Reply:
TY